components/compositing/compositor.rs

```diff
@@ -449,13 +449,11 @@ impl<Window: WindowMethods> IOCompositor<Window> {
         while self.port.try_recv_compositor_msg().is_some() {}
 
         // Tell the profiler, memory profiler, and scrolling timer to shut down.
-        match ipc::channel() {
-            Ok((sender, receiver)) => {
-                self.time_profiler_chan.send(time::ProfilerMsg::Exit(sender));
-                let _ = receiver.recv();
-            },
-            Err(_) => {},
+        if let Ok((sender, receiver)) = ipc::channel() {
+            self.time_profiler_chan.send(time::ProfilerMsg::Exit(sender));
+            let _ = receiver.recv();
```

Comment:
What the `let _ = ` is meant for? (NB: learning Rust, sorry if that question is silly :)).